dev/TreeView/TreeViewItem.cpp

```diff
@@ -331,7 +331,6 @@ void TreeViewItem::OnPropertyChanged(const winrt::DependencyPropertyChangedEvent
                 // The children have changed, validate and update GlyphOpacity
                 bool hasChildren = HasUnrealizedChildren() || treeViewNode->HasChildren();
                 GlyphOpacity(hasChildren ? 1.0 : 0.0);
-                UpdateNodeIsExpandedAsync(*treeViewNode, IsExpanded());
```

Comment:
Do you still need to do it in the case where the container is already prepared and then the source gets set on the node after that?

Reply:
I think PrepareContainerForItem will get called every time we change ItemsSource? There were some odd behaviors when keeping them both in place. I guess it caused some race conditions because the operation is async...